overlord/restart/restart_test.go

```diff
@@ -389,15 +389,24 @@ func (s *restartSuite) TestPendingForSystemRestart(c *C) {
 	chg3.AddTask(t7)
 	t6.WaitFor(t5)
 	t7.WaitFor(t5)
-	err = restart.FinishTaskWithRestart(t6, state.DoneStatus, restart.RestartSystem, "some-snap", nil)
+	err = restart.FinishTaskWithRestart(t5, state.DoneStatus, restart.RestartSystem, "some-snap", nil)
 	c.Assert(err, FitsTypeOf, &state.Wait{})
 	t6.SetStatus(state.WaitStatus)
 	t7.SetStatus(state.DoStatus)
-	c.Check(chg1.IsReady(), Equals, false)
+	c.Check(chg3.IsReady(), Equals, false)
+
+	chg4 := st.NewChange("pending", "...")
+	t8 := st.NewTask("wait-task", "...")
+	chg4.AddTask(t8)
```

Comment:
Can you please add a comment saying what status t8 is in, or add an explicit call to SetStatus?